### PR TITLE
Adds links from threat intel integrations guide to IoC page

### LINKED ISSUE #3824: [BUG] Threat intel creation guide to Intelligence dashboard and troubleshooting
**Description**: 
It would be good to set a link from threat intel setup integration guide:
https://www.elastic.co/guide/en/security/8.9/es-threat-intel-integrations.html
to indicators dashboard page:
https://www.elastic.co/guide/en/security/8.9/indicators-of-compromise.html
and in case the IoC are empty to troubleshooting page:
https://www.elastic.co/guide/en/security/8.9/indicators-of-compromise.html#troubleshoot-indicators-page

Could be possibly that if kibana setting `securitySolution:defaultIndex` is not maintained appropriately could be that:
Dashboards → Overview
Doesn't return the correct output in the Indicator sections as well.

### PR DESCRIPTION
**Pull request by natasha-moore-elastic**
- Resolves #3824 

Preview: [Enable threat intelligence integrations](https://security-docs_3857.docs-preview.app.elstc.co/guide/en/security/master/es-threat-intel-integrations.html)

### CODE REVIEW

**Review by nastasha-solomon — COMMENTED** [review]
Thanks for making these improvements, @natasha-moore-elastic! I'd also suggest getting a technical review from @dhru42 or @PhilippeOberti since they'll know for sure whether the troubleshooting steps for the Indicators page also apply to the [Threat Intelligence view](https://www.elastic.co/guide/en/security/8.9/overview-dashboard.html#_threat_intelligence) on the Overview dashboard

File: docs/getting-started/threat-intel-integrations.asciidoc [hunk 1]
```diff
@@ -41,7 +41,7 @@ If you know the name of {agent} integration you want to install, you can search
 
 =========================
 . Select an {agent} integration, then complete the installation steps.
-. Return to the Threat Intelligence view on the Overview page. Refresh the page if indicator data isn't displaying.
+. Return to the Threat Intelligence view on the Overview page. If indicator data isn't displaying, refresh the page or refer to <<troubleshoot-indicators-page>>.
```
[the remark this answers is not in the document]
Comment (natasha-moore-elastic) [reply docs/getting-started/threat-intel-integrations.asciidoc]: Thanks for the suggestions and the extra context, @nastasha-solomon!
Resolved in [fed2798](https://github.com/elastic/security-docs/pull/3857/commits/fed2798c9bffb57d2af569c2fa01cbbc2d54fcac).

**Comment by nastasha-solomon** [comment]
Sorry about that, @PhilippeOberti. I should've linked to the content that I was referring to. Here are the troubleshooting steps I'm talking about: https://www.elastic.co/guide/en/security/master/indicators-of-compromise.html#troubleshoot-indicators-page

These were written for troubleshooting data on the Indicators page, and I'm wondering if they also work for troubleshooting data issues with the Threat intelligence card on the Overview dashboard -- specifically the lack of data displaying in the card.

Screenshot 2023-09-07 at 10 55 13 AM

**Comment by PhilippeOberti** [comment]
> Sorry about that, @PhilippeOberti. I should've linked to the content that I was referring to. Here are the troubleshooting steps I'm talking about: https://www.elastic.co/guide/en/security/master/indicators-of-compromise.html#troubleshoot-indicators-page
> 
> These were written for troubleshooting data on the Indicators page, and I'm wondering if they also work for troubleshooting data issues with the Threat intelligence card on the Overview dashboard -- specifically the lack of data displaying in the card.

I would assumme the steps are identical yes, but I'm not 100% sure as I haven't worked on the dashboard at all

**Comment by nastasha-solomon** [comment]
No worries and thanks, @PhilippeOberti. 

@nkhristinin @rylnd do you happen to know if the troubleshooting tips [here](https://www.elastic.co/guide/en/security/master/indicators-of-compromise.html#troubleshoot-indicators-page) can also be used on the Threat intelligence card (on the Overview dashboard) when data's not displaying?

**Comment by rylnd** [comment]
@nastasha-solomon yep, it's the same underlying data, so those tips apply there as well 👍

**Comment by nastasha-solomon** [comment]
Thanks for confirming, @rylnd. @natasha-moore-elastic we can go ahead with updating the referenced section then. Feel free to use my suggestion [here](https://github.com/elastic/security-docs/pull/3857#discussion_r1316157192) or update as you see fit. TY!

**Pull request merged**